src/backend/distributed/transaction/transaction_management.c

```diff
@@ -312,6 +312,8 @@ CoordinatedTransactionCallback(XactEvent event, void *arg)
 			break;
 		}
 	}
+
+	UnsetCitusNoticeLevel();
```

Comment:
We only need to do this on abort, since otherwise we would have followed the normal flow, so maybe put it at line 240.

Reply:
and also in the `SUBXACT_EVENT_ABORT_SUB` handler